.travis.yml

```diff
@@ -29,7 +29,7 @@ notifications:
 env:
   global:
     # Reduce the memory pressure on Travis CI to reduce build failures.
-    - GRADLE_OPTS="-Xms128m"
```

Comment:
I think this is not going to help. Try to run rebuild multiple times.

Reply:
Thank you! I'll try that!

Reply:
lets not change this. The tests have some flakiness due to which they fail sometimes.

Reply:
I was looking into how we might be able to fix this, but I didn't get very far. We're running into many different issues be it OOM or otherwise. Will leave it as it currently is for now. Thank you!